wiki/subscription-with-email-notification.md

```diff
@@ -44,6 +44,38 @@ The form value will be run through JMESPath engine so it is possible to use
 JMESPath expressions to extract content from the aggregated object. The 
 form value can only be one JSON object.
 
+Below are the limitations of notificationMessageKeyValues.
+
+### Limitations
+
+* The content should be in JSON format.
```

Comment:
I like this bullet list, it's easier to find what the reader is looking for. You can trim down or remove the above paragraph and just keep the bullet list I think. 

Remember to add another bullet on the form key.

Reply:
Done the changes as suggested